### DISCUSSION TITLE
1.4.1-beta0

### DISCUSSION THREAD

**Discussion by jglev** [jglev/obsidian-carry-forward#17] [Announcements]
- Fixed bug whereby when copying multiple lines each without a block ID , only block links (without the content of the lines) would get copied for the first and last lines.

<hr /><em>This discussion was created from the release <a href='https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.1-beta0'>1.4.1-beta0</a>.</em>

**Comment by FelipeRearden** [comment]
Hello @jglev !!!!!!

Just to let you know that I repeat my test cycle with this new beta release 1.4.1-beta0 and from my tests 100% working :)

I wish you a fantastic day!!!!!!!!!

**Reply by jglev** [reply]
Thank you, @FelipeRearden! I appreciate knowing that, and appreciate your eyes on it!
